solution/ui/e2e/cypress/schemas/example-schema.js

```diff
@@ -1,3 +1,5 @@
+// EXAMPLE SCHEMA
+
```

Comment:
We're no longer using this file -- we're not validating any response schema anymore, since we only needed to do that for third party responses like Search.gov which we're no longer doing.  However, I can see us making calls to third party services in the future and I'd hate to have to search through the git logs to reproduce this work.  As such, I'm leaving it here for now as an example of how to create a schema to validate.